src/lib/utils/date.ts

```diff
@@ -257,6 +257,36 @@ export const deriveFormattedTimestamp = (datetime) => {
   return reformatTime(formattedTime)
 }
 
+import { format } from 'date-fns'
+
+interface DateTimeRange {
+  value: number
+  endValue: number
+  timezone?: string
+}
```

Comment:
Is this the same shape as in `vets-website`? If not, I recommend renaming `value` and `endValue` to `startTime` and `endTime`. In either case, I recommend putting a JSDoc comment on the properties to describe them.
```suggestion
/**
 * Bonus points for describing this object, but the name is pretty descriptive by itself. 🤷‍♂️
 */
interface DateTimeRange {
  /**
   * Unix timestamp for the beginning of the date time range.
   */
  value: number
  /**
   * Unix timestamp for the end of the date time range.
   */
  endValue: number
  /**
   * ISO timezone string (e.g. 'America/Los_Angeles')
   */
  timezone?: string
}
```

It feels pretty verbose, but we get to leverage the power of our language server hover docs that way. 🤓